types/koa-helmet/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for koa-helmet 3.1
+// Project: https://github.com/venables/koa-helmet#readme
+// Definitions by: Nick Simmons <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import {
+    IHelmetConfiguration,
+    IHelmetFrameguardConfiguration,
+    IHelmetHstsConfiguration,
+    IHelmetXssFilterConfiguration,
+    IHelmetDnsPrefetchControlConfiguration,
+    IHelmetHpkpConfiguration,
+    IHelmetReferrerPolicyConfiguration } from 'helmet';
+import {Middleware, Context} from 'koa';
+
+export type KoaHelmetContentSecurityPolicyDirectiveFunction = (ctx: Context) => string;
+
+export type KoaHelmetCspDirectiveValue = string | KoaHelmetContentSecurityPolicyDirectiveFunction;
+
+export interface KoaHelmetContentSecurityPolicyDirectives {
+    baseUri?: KoaHelmetCspDirectiveValue[];
+    childSrc?: KoaHelmetCspDirectiveValue[];
+    connectSrc?: KoaHelmetCspDirectiveValue[];
+    defaultSrc?: KoaHelmetCspDirectiveValue[];
+    fontSrc?: KoaHelmetCspDirectiveValue[];
+    formAction?: KoaHelmetCspDirectiveValue[];
+    frameAncestors?: KoaHelmetCspDirectiveValue[];
+    frameSrc?: KoaHelmetCspDirectiveValue[];
+    imgSrc?: KoaHelmetCspDirectiveValue[];
+    mediaSrc?: KoaHelmetCspDirectiveValue[];
+    objectSrc?: KoaHelmetCspDirectiveValue[];
+    pluginTypes?: KoaHelmetCspDirectiveValue[];
+    reportUri?: string;
+    sandbox?: KoaHelmetCspDirectiveValue[];
+    scriptSrc?: KoaHelmetCspDirectiveValue[];
+    styleSrc?: KoaHelmetCspDirectiveValue[];
+}
+
+export interface KoaHelmetContentSecurityPolicyConfiguration {
+    reportOnly?: boolean;
+    setAllHeaders?: boolean;
+    disableAndroid?: boolean;
+    browserSniff?: boolean;
+    directives?: KoaHelmetContentSecurityPolicyDirectives;
+}
+
+export interface KoaHelmet {
+    (options?: IHelmetConfiguration): Middleware;
+    contentSecurityPolicy(options?: KoaHelmetContentSecurityPolicyConfiguration): Middleware;
+    dnsPrefetchControl(options?: IHelmetDnsPrefetchControlConfiguration): Middleware;
+    frameguard(options?: IHelmetFrameguardConfiguration): Middleware;
+    hpkp(options?: IHelmetHpkpConfiguration): Middleware;
+    hsts(options?: IHelmetHstsConfiguration): Middleware;
+    ieNoOpen(): Middleware;
+    noCache(options?: any): Middleware;
+    noSniff(): Middleware;
+    referrerPolicy(options?: IHelmetReferrerPolicyConfiguration): Middleware;
+    xssFilter(options?: IHelmetXssFilterConfiguration): Middleware;
+}
+
+declare const koaHelmet: KoaHelmet;
+export default koaHelmet;
```

Comment:
use `export = koaHelmet` instead. See  https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default